extensions/ql-vscode/src/common/commands.ts

```diff
@@ -35,6 +35,7 @@ export type SingleSelectionCommandFunction<Item> = (
 // Builtin commands where the implementation is provided by VS Code and not by this extension.
 // See https://code.visualstudio.com/api/references/commands
 export type BuiltInVsCodeCommands = {
+  "codeQLDatabases.focus": () => Promise<void>;
```

Comment:
Perhaps add a comment for this command? It's not entirely obvious if you're just looking at the file that this is a command added by VS Code on custom views.